docs/docs/rad-lab-ui/ui_deployment/deploy-a-module.md

```diff
@@ -8,6 +8,18 @@ This is the primary flow and purpose for RAD Lab UI. A step-by-step graphical in
 
 Admins and Users are both able to deploy modules.
 
+Below video provides a quick overview on how to setup RAD Lab UI and deploy RAD Lab modules with the same.
```

Comment:
The video below provides a quick overview on how to setup RAD Lab UI variables and deploy RAD Lab modules.

Reply:
Updated verbose.